src/ipumspy/api/metadata.py

```diff
@@ -34,15 +45,15 @@ class DatasetMetadata(IpumsMetadata):
 
     Args:
         collection: name of an IPUMS data collection
-        name: Name of an IPUMS dataset associated with the indicated collection
+        name: Name of an IPUMS dataset included in the collection
     """
 
     collection: str
-    """name of an IPUMS data collection"""
+    """name of an IPUMS data collection. Currently only available for IPUMS NHGIS"""
     name: str
-    """IPUMS NHGIS dataset name"""
+    """IPUMS dataset name from an IPUMS aggregate data collection"""
     nhgis_id: Optional[str] = field(default=None, init=False)
```

Comment:
I don't love having attributes with collection-specific names. Since it looks like these are actually the names of fields in the extract request schema, making this more general at this point might be premature optimization, but if other aggregate data collections were to have similar concepts, this could get ugly. And if that situation were to arise, it is probably best to make that change server side anyway.... but just something to keep an eye on as API offerings expand.